src/_listBase.scss

```diff
@@ -165,6 +165,14 @@
     color: var(--sapList_FooterTextColor);
     font-size: $fd-list-normal-font-size;
     height: 2rem;
+
+    &--right {
+      float: right;
```

Comment:
Floats are quite old, don't use them in styles lib
![image](https://user-images.githubusercontent.com/26483208/97295341-70ddd000-184f-11eb-91f1-f2c902c5f539.png)
I believe you can do it with flexbox